helix-core/src/main/java/org/apache/helix/util/HelixUtil.java

```diff
@@ -173,10 +174,43 @@ public static String serializeByComma(List<String> objects) {
    * @param resourceConfigs
    * @return
    */
-  public static Map<String, ResourceAssignment> getIdealAssignmentForWagedFullAuto(
+  public static Map<String, ResourceAssignment> getTargetAssignmentForWagedFullAuto(
```

Comment:
Shouldn't we leave this unchanged since this is a public method?

Reply:
That's my initial thought too, which is why I kept the method name in my initial commit. 

However, keeping this name makes it hard to differentiate from the other method (both are "ideal" assignments), and I liked Jiajun's new names. The only current reference is the migration testing tool, which I think is fine to be updated as Helix version is updated.